cloud-chatops/src/features/base_feature.py

```diff
@@ -115,6 +115,15 @@ def _format_prs(prs: List[PrData]) -> List[PrData]:
             formatted_prs.append(PRMessageBuilder().check_pr(pr))
         return formatted_prs
 
+    def validate_user(self, user: str) -> None:
+        """Validate that the given user is in the workspace."""
+        try:
+            self.client.users_profile_get(user=user)
+        except SlackApiError as exc:
+            raise UserNotFound(
+                f"The user with member ID {user} is not in this workspace."
+            ) from exc
+
```

Comment:
Codecov has raised a warning about a missing unit test, could we add a unit test to check this error is raised?

Reply:
That is covered [here](https://github.com/stfc/cloud-docker-images/blob/4ac5e6de365e2713e8d90939027358f1ebf5a280/cloud-chatops/tests/test_base_feature.py#L178) I'm not sure why code cov is complaining. @DavidFair do you have any ideas?